Job: Docker-PRs [FAILED](https://jenkins.dockerproject.org/job/Docker-PRs/23956/console):

~~~console

---
s/1.11.0-dev/validate-default-seccomp)

---> Making bundle: validate-gofmt (in bundles/1.11.0-dev/validate-gofmt)
Congratulations!  All Go source files are properly formatted.

---> Making bundle: validate-lint (in bundles/1.11.0-dev/validate-lint)
Errors from golint:
docker/listeners/listeners_unix.go:23:27: error strings should not end with punctuation

Please fix the above errors. You can test via "golint" and commit the result.

Build step 'Execute shell' marked build as failure
[PostBuildScript] - Execution post build scripts.
[Docker-PRs] $ /bin/sh -xe /tmp/hudson6001630994105657457.sh
+ set +x


---

~~~

--- a/docker/listeners/listeners_unix.go
+++ b/docker/listeners/listeners_unix.go
@@ -59,7 +59,7 @@ func listenFD(addr string, tlsConfig *tls.Config) ([]net.Listener, error) {
 	}
 
 	if len(listeners) == 0 {
-		return nil, fmt.Errorf("No sockets found")
+		return nil, fmt.Errorf("No sockets found. Make sure the docker daemon was started by systemd.")
 	}
 
 	// default to all fds just like unix:// and tcp://

